CRM/EventsExtras/Upgrader/Base.php

```diff
@@ -1,15 +1,14 @@
 <?php
 
 // AUTO-GENERATED FILE -- Civix may overwrite any changes made to this file
-use CRM_EventsExtras_ExtensionUtil as E;
 
 /**
  * Base class which provides helpers to execute upgrade logic
  */
 class CRM_EventsExtras_Upgrader_Base {
 
   /**
-   * @var varies, subclass of this
+   * @var variessubclassofthis
```

Comment:
Nasty, but I know this class is auto-generated... so could leave as is. I'm guessing phpcs was complaining?

Reply:
@MiyaNoctem  I did run phpcbf on it but will be excluded from next commit.